.github/CONTRIBUTING.md

```diff
@@ -4,72 +4,149 @@ Water.css becomes better for everyone when people like you help make it better!
 
 Have any questions or concerns? Did I forget an element or selector? Does something look ugly? Feel free to submit an issue or pull request.
 
-Before contributing, please read the [code of conduct](CODE_OF_CONDUCT.md). Also you agree that your contributions will be licensed under its [MIT License](./LICENSE.md).
+Before contributing, please read the [code of conduct](CODE_OF_CONDUCT.md). Also, you agree that your contributions will be licensed under this project's [MIT License](../LICENSE.md).
 
-## How to get started
+## Overview
 
-1. Get a copy of repository. It is recommended to fork it first and clone to your machine using `git`.
+Please take a moment to read through the following guidelines:
 
-2. Make sure that you have [yarn](https://yarnpkg.com) and install dependencies listed in `package.json` using it.
+- [Get started](#get-started)
+- [Find issues to work on](#find-issues-to-work-on)
+- [Add or change styles](#add-or-change-styles)
+- [Create your pull request](#create-your-pull-request)
+- [Project structure](#project-structure)
 
-```
-yarn
-```
+<br>
 
-3. Then you can run development server with live reloading out of the box and play around with the framework.
+> **TL;DR**:
+> 1. `yarn install` – install dependencies
+> 2. `yarn dev` – start the dev server, make your changes
+> 3. `yarn validate` – make sure your changes pass all tests
+> 4. `yarn changeset` – describe your changes for the changelog
```

Comment:
When I run `yarn changeset`, it lets me know that the current version is 1.4.0. Not sure where that's defined, but it may be something to change after release?

Reply:
It's defined in the package.json - we have to write a changeset before merging.